ios/ReactNativeBrownfield.swift

```diff
@@ -136,6 +143,25 @@ class ReactNativeBrownfieldDelegate: RCTDefaultReactNativeFactoryDelegate {
     }
   }
   
+  /**
+   * Stops React Native and releases the underlying factory instance.
+   */
+  @objc public func stopReactNative() {
+    if !Thread.isMainThread {
+      DispatchQueue.main.async { [weak self] in self?.stopReactNative() }
+      return
+    }
+
+    guard let factory = reactNativeFactory else { return }
+
+    factory.bridge?.invalidate()
+
+    NotificationCenter.default.removeObserver(self)
+    onBundleLoaded = nil
+
+    reactNativeFactory = nil
```

Comment:
Did you test the scenario of what happens when react native view is still shown while calling this? Let's make sure it doesn't crash the app.

Reply:
Maybe there is a way to detect current react native surfaces in the app and show a warning if the user tries to do this with views still on screen